src/api/app/views/webui2/shared/_requests_state_and_type_form.html.haml

```diff
@@ -0,0 +1,16 @@
+.row
+  .form-group
+    .col-sm-12
+      = label_tag('request_type_select', 'Display', class: 'control-label')
+      = select_tag('request_type_select',
+                   options_for_select(Webui::RequestHelper::AVAILABLE_TYPES, selected_type),
+                   data: { table: 'all_requests_table' },
+                   class: 'form-control custom-select custom-select-sm form-control-sm dataTables_length')
+.row
+  .form-group
+    .col-sm-12
+      = label_tag('request_state_select', 'Requests in state', class: 'control-label')
+      = select_tag('request_state_select',
+                   options_for_select(Webui::RequestHelper::AVAILABLE_STATES, selected_state),
+                   data: { table: 'all_requests_table' },
+                   class: 'form-control custom-select custom-select-sm form-control-sm dataTables_length')
```

Comment:
Now this looks a bit strange on the larger screen. 

![screenshot from 2018-08-22 17-13-20](https://user-images.githubusercontent.com/514785/44472742-11605600-a62f-11e8-8df1-4ed38e87d767.png)

How about you move the selects into partials and then you can render them more than one time and [hide](https://getbootstrap.com/docs/4.1/utilities/display/#hiding-elements) what you don't need in the different breakpoints.

Reply:
Updated. I found the CSS for setting correct breakpoints for different screensizes. So no need for hiding and showing elements:tada: